Agents/WeatherAgent/docker-compose.yml.template

```diff
@@ -10,7 +10,7 @@ services:
         condition: none
     environment:
       - STACK_NAME=${STACK_NAME}
-      - API_KEY=
+      - API_KEY=21db8f6413957d7fdb6cfd98e05e03eb
```

Comment:
Are you sure you want to commit this? I'm guessing not... Also, what is desperately needed here is some documentation, probably in the main Readme file of this agent (which doesn't even exist!) that the user needs to add an API key here before attempting to build the agent, together with an explanation of what API this is meant to be, ideally with a web-link where to obtain such a key.

Reply:
It's in the Setup section of the readme, the key will be removed